sklearn/utils/sparsefuncs_fast.pyx

```diff
@@ -278,7 +278,10 @@ def inplace_csr_row_normalize_l1(X):
     _inplace_csr_row_normalize_l1(X.data, X.shape, X.indices, X.indptr)
 
 
-def _inplace_csr_row_normalize_l1(np.ndarray[floating, ndim=1] X_data, shape,
+@cython.boundscheck(False)
+@cython.wraparound(False)
+@cython.cdivision(True)
```

Comment:
@rvraghav93 This is also documented here (http://docs.cython.org/src/reference/compilation.html#compiler-directives)